### PR TITLE
fill in more info starting from source_id 155 downwards

### PR DESCRIPTION
**Pull request by vorugantia**
See in-line comments for where I found all of the information. A "?" means I was unsure where to get this information from.

Can you make sure I wrote all of the correct information? Am I getting everything from the right places? Also, what should I put in "gamma_names" and where should I look for the information? What about papers? I looked at the sources you did and I don't think you're using all of the papers listed on tevcat.

### CODE REVIEW

File: input/sources/tev-000155.yaml [hunk 1]
```diff
@@ -1,11 +1,25 @@
 source_id: 155
 
+common_name: H 2356-309 # "canonical name" row on tevcat website. Also listed in tevcat.ecsv, "source_name" column
+gamma_names: [] # ?
+other_names: [1H 2354-315, HESS J2359-306] # "Other names" row on tevcat's website. Also listed in "other_names" column in tgevcat.ecsv
```
Comment (cdeil) [inline input/sources/tev-000155.yaml]: Put the HESS name in `gamma_names` instead.
`gamma_names` should be for sources detected by gamma-ray instruments, i.e. with prefix HESS, VER (VERITAS) or MAGIC or HAWC.
It's OK to leave this empty for sources where you don't have such a name.

**Comment by cdeil** [comment]
@vorugantia - Ready to merge?

**Comment by cdeil** [comment]
I'm merging this now.

Thanks!

I wasn't sure if you had further edits / commits.
If you do, please make another PR.

Otherwise, tomorrow, I'll finish up filling the info for all sources.

**Comment by vorugantia** [comment]
Yeah sorry, I forgot to tell you. I might have done a couple more sources but I'll open a new PR.

**Pull request merged**